Frontend/ui-driver/src/Screens/BookingOptionsScreen/View.purs

```diff
@@ -122,40 +121,44 @@ downgradeVehicleView push state =
           [ width MATCH_PARENT
           , height WRAP_CONTENT
           , margin (Margin 16 0 16 20)
-          , text $ (getString DOWNGRADING_VEHICLE_WILL_ALLOW_YOU_TO_TAKE_BOTH_1) 
-                    <> " " <> downgradeFrom <> " "
-                    <> (getString DOWNGRADING_VEHICLE_WILL_ALLOW_YOU_TO_TAKE_BOTH_2)
-                    <> " " <> downgradeTo <> " "
-                    <> (getString DOWNGRADING_VEHICLE_WILL_ALLOW_YOU_TO_TAKE_BOTH_3)
+          , text $ (getString DOWNGRADING_VEHICLE_WILL_ALLOW_YOU_TO_TAKE_BOTH_1) <> getDowngradeOptionsText state.data.vehicleType
```

Comment:
`DOWNGRADING_VEHICLE_WILL_ALLOW_YOU_TO_TAKE_BOTH_1 ` should also be included in `getDowngradeOptionsText ` function